tests/_support/ea-server-mocker/src/Options_Page.php

```diff
@@ -37,39 +37,42 @@ public function register_settings() {
 
 	public function render() {
 		?>
-		<div class="wrap">
-			<h1>Event Aggregator Server Mocker Settings</h1>
+        <div class="wrap">
+            <h1>Event Aggregator Server Mocker Settings</h1>
```

Comment:
This whole chunk looks like it was un-tabbed by one level accidentally. Tab it back?